provider/online.go

```diff
@@ -125,15 +94,19 @@ func (store *localOnlineStore) AsOnlineStore() (OnlineStore, error) {
 func (store *localOnlineStore) GetTable(feature, variant string) (OnlineStoreTable, error) {
 	table, has := store.tables[tableKey{feature, variant}]
 	if !has {
-		return nil, &TableNotFound{feature, variant}
+		wrapped := fferr.NewDatasetNotFoundError(feature, variant, nil)
+		wrapped.AddDetail("provider", store.ProviderType.String())
+		return nil, wrapped
 	}
 	return table, nil
 }
 
 func (store *localOnlineStore) CreateTable(feature, variant string, valueType ValueType) (OnlineStoreTable, error) {
 	key := tableKey{feature, variant}
 	if _, has := store.tables[key]; has {
-		return nil, &TableAlreadyExists{feature, variant}
+		wrapped := fferr.NewDatasetAlreadyExistsError(feature, variant, nil)
+		wrapped.AddDetail("provider", store.ProviderType.String())
```

Comment:
it's not possible for store.ProviderType to be nil right?

Reply:
It's possible inasmuch as we don't validate this, but it shouldn't happen.